queries.sql

```diff
@@ -21,3 +21,27 @@ select * from animals where name != 'Gabumon';
 
 -- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
 select * from animals where weigth_kg between 10.3 and 17.4;
+
+
+-- How many animals are there?
+select count(*) from animals; 
+
+-- How many animals have never tried to escape?
+select count(*) from animals where escape_attempts = 0; 
+
+-- What is the average weight of animals?
+select avg(weigth_kg) from animals;
+
+-- Who escapes the most, neutered or not neutered animals?
+select name, escape_attempts from animals 
+group by name, escape_attempts 
+order by escape_attempts DESC
+limit 1;
+
```

Comment:
- [OPTIONAL ] Who escapes the most, neutered or not neutered animals? this one should be:

  ```sql
     SELECT neutered, AVG(escape_attempts) FROM animals
     GROUP BY neutered;
  ```

   or

  ```sql
    SELECT neutered, MAX(escape_attempts) FROM animals
    GROUP BY neutered;
  ```

Reply:
Thank you very much for your clarification. I think I misunderstood the question. I found the name of the one who escapes the most Now I understand why it was harder to do, but It lead me to investigate and learn new things like order by and limit.